bectechnologies.md

```diff
@@ -0,0 +1,100 @@
+## BEC Technologies, Inc.
+<a href="www.bectechnologies.net">
+<img src="https://bectechnologies.net/wp-content/uploads/2018/11/BEC-logo-200x72.png" alt="BEC Logo" width="200" height="72"></A>
+ 
+## Summary
+BEC Technologies is a leading developer and manufacturer of 4G LTE and 5G wireless broadband networking solutions for Telcos, mobile carriers, (WISPs) wireless internet providers, Enterprise,  Industrial, and many other markets. BEC's comprehensive product portfolio of solutions incorporates Fixed Data Routers, VoIP/VoLTE Gateways, Rugged Outdoor, Industrial/M2M Connectivity, Public Safety,  Fleet/Telematics, and Cloud-based remote device management. Our solutions are designed specifically for high availability, reliability, and secure connectivity, all backed up with class-leading technical service and support.
+
+BEC is Managing Millions of Connected Devices Worldwide, driving the global transformation to a connected world!  For more information, please visit <a href="http://www.bectechnologies.net/">www.bectechnologies.net</a> or follow us on Twitter @BECTechnologies.
+
+## Company Information (required)
+* BEC Technologies, Inc.
+* Founded in 2004 (parent company Billion Electric Co. Ltd., TSE: #3027, Founded- 1973)
+* Market Leadership, #1 in the US for LTE Fixed Wireless Access IFWA)
+* Global Brand with distribution to over 100 countries and territories
+* Millions of deployed & approved with 350+ operators
+* We are dedicated to simplifying wireless broadband connectivity and offering solutions to 
+  meet the growing demands of varying broadband market segments.
+ 
+
+## Product Information (required)
+The BEC Helium miner is full hotspot, intended for outdoor use.  Our Outdoor Hotpot will include LoRA, Wi-Fi, 
```

Comment:
The BEC Helium miner is a Light Hotspot, intended for outdoor use.   Our Outdoor Hotspot will include LoRA, Wi-Fi,